website/source/docs/providers/aws/r/api_gateway_deployment.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "aws"
+page_title: "AWS: aws_api_gateway_deployment"
+sidebar_current: "docs-aws-resource-api-gateway-deployment"
+description: |-
+  Provides an API Gateway Deployment.
+---
+
+# aws\_api\_gateway\_deployment
+
+Provides an API Gateway Deployment.
+
+**Warning:** Depends on a completely setup `aws_api_gateway_rest_api`. To ensure this
```

Comment:
1) The convention (which also turns this paragraph into a nice blue box) is
```
-> **Note:** yada yada
```

2) As discussed previously, it does depend on `aws_api_gateway_method`, doesn't it?